### ISSUE TITLE
missing flag value error while creating new profile

### ISSUE THREAD

**Issue by sevagas** [BishopFox/sliver#746]
**Description**
I cannot create profiles. I tried also on clean version.
On an up to date install (v1.5.18)
I get the bug: "panic: missing flag value: flag 'name' not registered"

**To Reproduce**
`sliver > profiles new --mtls 192.168.122.1 --skip-symbols --format shellcode win-shellcode`

Sliver crashes with next output:
`panic: missing flag value: flag 'name' not registered

goroutine 1 [running]:
github.com/desertbit/grumble.FlagMap.String(0x1?, {0x1271cb9, 0x4})
        github.com/desertbit/grumble@v1.1.1/flagmap.go:58 +0x105
github.com/bishopfox/sliver/client/command/generate.parseCompileFlags(0xc000304060, 0xc00053a080)
        github.com/bishopfox/sliver/client/command/generate/generate.go:167 +0x51
github.com/bishopfox/sliver/client/command/generate.ProfilesNewCmd(0xc000304060, 0xc00053a080)
        github.com/bishopfox/sliver/client/command/generate/profiles-new.go:31 +0x53
github.com/bishopfox/sliver/client/command.BindCommands.func141(0xc0009bc1e0?)
        github.com/bishopfox/sliver/client/command/commands.go:1493 +0x45
github.com/desertbit/grumble.(*App).RunCommand(0xc00079c000, {0xc0003a0080?, 0x51?, 0x19844c8?})
        github.com/desertbit/grumble@v1.1.1/app.go:273 +0x19f
github.com/desertbit/grumble.(*App).runShell(0xc00079c000)
        github.com/desertbit/grumble@v1.1.1/app.go:485 +0x4c6
github.com/desertbit/grumble.(*App).RunWithReadline(0xc00079c000, 0xc00000e6f0)
        github.com/desertbit/grumble@v1.1.1/app.go:407 +0x53f
github.com/desertbit/grumble.(*App).Run(0xc00079c000)
        github.com/desertbit/grumble@v1.1.1/app.go:291 +0x11a
github.com/bishopfox/sliver/client/console.Start({0x19ca370?, 0xc00036c150}, 0x1548358, 0x1547d58, 0x0)
        github.com/bishopfox/sliver/client/console/console.go:142 +0x537
github.com/bishopfox/sliver/client/cli.StartClientConsole()
        github.com/bishopfox/sliver/client/cli/cli.go:100 +0x1a6
github.com/bishopfox/sliver/client/cli.glob..func1(0x20c0f20?, {0x12719ed?, 0x0?, 0x0?})
        github.com/bishopfox/sliver/client/cli/cli.go:74 +0x76
github.com/spf13/cobra.(*Command).execute(0x20c0f20, {0xc00003c330, 0x0, 0x0})
        github.com/spf13/cobra@v1.1.1/command.go:854 +0x663
github.com/spf13/cobra.(*Command).ExecuteC(0x20c0f20)
        github.com/spf13/cobra@v1.1.1/command.go:958 +0x39c
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.1.1/command.go:895
github.com/bishopfox/sliver/client/cli.Execute()
        github.com/bishopfox/sliver/client/cli/cli.go:105 +0x25
main.main()
      github.com/bishopfox/sliver/client/main.go:43 +0x17

`

**Expected behavior**
The profile should be created.

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Version 18.04

**Comment by moloch--** [comment]
Also seems to affect `generate`

**Comment by drdsec** [comment]
Seeing a similar issue here -

[server] sliver > profiles new --mtls 192.168.0.114:443 --skip-symbols --format shellcode win-shellcode

stacktrace from panic:
goroutine 1 [running]:
runtime/debug.Stack()
        runtime/debug/stack.go:24 +0x65
github.com/bishopfox/sliver/server/cli.glob..func3.1()
        github.com/bishopfox/sliver/server/cli/cli.go:120 +0x72
panic({0x12599a0, 0xc0003467f0})
        runtime/panic.go:838 +0x207
github.com/desertbit/grumble.FlagMap.String(0x1?, {0x1400d08, 0x4})
        github.com/desertbit/grumble@v1.1.1/flagmap.go:58 +0x105
github.com/bishopfox/sliver/client/command/generate.parseCompileFlags(0xc0000a2120, 0xc000604580)
        github.com/bishopfox/sliver/client/command/generate/generate.go:167 +0x51
github.com/bishopfox/sliver/client/command/generate.ProfilesNewCmd(0xc0000a2120, 0xc000604580)
        github.com/bishopfox/sliver/client/command/generate/profiles-new.go:31 +0x53
github.com/bishopfox/sliver/client/command.BindCommands.func141(0xc000b40000?)
        github.com/bishopfox/sliver/client/command/commands.go:1493 +0x45
github.com/desertbit/grumble.(*App).RunCommand(0xc000496380, {0xc000186000?, 0x55?, 0xa75a310?})
        github.com/desertbit/grumble@v1.1.1/app.go:273 +0x19f
github.com/desertbit/grumble.(*App).runShell(0xc000496380)
        github.com/desertbit/grumble@v1.1.1/app.go:485 +0x4c6
github.com/desertbit/grumble.(*App).RunWithReadline(0xc000496380, 0xc0003fc4c8)
        github.com/desertbit/grumble@v1.1.1/app.go:407 +0x53f
github.com/desertbit/grumble.(*App).Run(0xc000496380)
        github.com/desertbit/grumble@v1.1.1/app.go:291 +0x11a
github.com/bishopfox/sliver/client/console.Start({0xa7a8630?, 0xc00037fff0}, 0xa2e0348, 0xa2e0810, 0x1)
        github.com/bishopfox/sliver/client/console/console.go:142 +0x537
github.com/bishopfox/sliver/server/console.Start()
        github.com/bishopfox/sliver/server/console/console.go:61 +0x3d8
github.com/bishopfox/sliver/server/cli.glob..func3(0xafa5320?, {0x1400a3c?, 0x0?, 0x0?})
        github.com/bishopfox/sliver/server/cli/cli.go:138 +0xbb
github.com/spf13/cobra.(*Command).execute(0xafa5320, {0xc0001122f0, 0x0, 0x0})
        github.com/spf13/cobra@v1.1.1/command.go:854 +0x663
github.com/spf13/cobra.(*Command).ExecuteC(0xafa5320)
        github.com/spf13/cobra@v1.1.1/command.go:958 +0x39c
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.1.1/command.go:895
github.com/bishopfox/sliver/server/cli.Execute()
        github.com/bishopfox/sliver/server/cli/cli.go:145 +0x25
main.main()
        github.com/bishopfox/sliver/server/main.go:43 +0x17

**Comment by moloch--** [comment]
fixed in v1.5.19, @sevagas note you'll need to add the profile name on the end of your command:

For example:
```
profiles new --os windows --arch 64bit --evasion --mtls 192.168.1.22:443 --skip-symbols --format shellcode asdf
```

Will create a profile named `asdf`
